include/swift/AST/Decl.h

```diff
@@ -4788,8 +4788,12 @@ enum class PropertyWrapperSynthesizedPropertyKind {
 /// VarDecl - 'var' and 'let' declarations.
 class VarDecl : public AbstractStorageDecl {
   friend class NamingPatternRequest;
+  friend class GetFieldIndexRequest;
```

Comment:
`GetFieldIndexRequest` doesn't appear to be defined

Reply:
good catch. I'll remove it